internal/nexodus/ctlserver_unix.go

```diff
@@ -41,9 +43,17 @@ func (ax *Nexodus) CtlServerLinuxStart(ctx context.Context, wg *sync.WaitGroup)
 	})
 }
 
+func UnixSocketPath() string {
+	if runtime.GOOS == Darwin.String() {
+		return unixSocketPathDarwin
```

Comment:
this could be a platform specific function.

Reply:
Yeah … but it doesn’t look like it’d make a meaningful difference to the binary size. I find this easier to follow than having to jump between files for this simple case. 

It’s probably worth a short design doc that lays out guidelines for the platform specific builds. I wouldn’t think it’s “make platform specific functions any time there is platform specific behavior,” is it? How about “split them when the implementation differs between platforms in non trivial ways and especially if platforms bring in different dependencies as this may have a meaningful  impact on binary size.”